src/elements/content-uploader/ContentUploader.js

```diff
@@ -893,7 +894,11 @@ class ContentUploader extends Component<Props, State> {
         }
 
         item.progress = Math.min(Math.round((event.loaded / event.total) * 100), 100);
-        item.status = STATUS_IN_PROGRESS;
+        if (item.progress === 100) {
+            item.status = STATUS_STAGED;
```

Comment:
It looks like this can be simplified to a ternary. Should this be status be added to the guard check on line 892?

Reply:
you're right, thanks!